portfolio/src/main/webapp/script.js

```diff
@@ -12,23 +12,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// A list of activities that could appear when clicking activities button.
-const activities = ['Coding 💻', 'Travelling 🗼', 'Brewing coffee ☕', 'Going on a nature walk 🌲'];
+// This is a list of passions to iterate through on the homepage
+const passions = ["Front-End Development?", "Machine Learning?", "Software Design?", "Technology Consulting?" ,"Data Science?", "Algorithms?", "User Experience?"];
+const typewriterLetterDelay = 100;
+const typewriterWordDelay = 1000;
 
+// Get the selector for the Passions type-writer function when the window loads.
+window.onload = function () { 
+    var passionSelector = document.getElementById("passions").childNodes[0]
```

Comment:
I assume you want to select the empty text node before the blinking cursor? It would be more readable to have an empty `span` that you explicitly select.

Reply:
100% agreed - fixed